### ISSUE TITLE
Declared locale identifier isn't reflected in outputted HTML

### ISSUE THREAD

**Issue by phieri** [upptime/upptime#1005]
**Describe the bug**
The declared locale identifier isn't reflected in outputted HTML. Pages are hardcoded as always English.

**To Reproduce**
1. Install and setup Upptime according to [instructions](https://upptime.js.org/docs/get-started).
2. Check what declared language is in HTML. The second row reads `<html lang=en>`.
3. Add non-English translation, including locale name, to configuration according to [instructions](https://upptime.js.org/docs/configuration#internationalization).
4. Again check declared language in HTML, it's still `lang=en`.

**Expected behavior**
The declared language should be used as HTML page language.

**Desktop**
 - OS: macOS 14.6.1
 - Browser: Safari 17.6

**Additional context**
- https://github.com/upptime/status-page/blob/54c2ff5a3d998d525ee4c7e68dc7ce7414d89c33/src/template.html#L2
- https://github.com/upptime/status-page/blob/54c2ff5a3d998d525ee4c7e68dc7ce7414d89c33/i18n.yml#L61

**Comment by AnandChowdhary** [comment]
You're right, this is my mistake, it's hardcoded to "en" in this line - https://github.com/upptime/status-page/blob/54c2ff5a3d998d525ee4c7e68dc7ce7414d89c33/src/template.html#L2. I would love to accept a PR for a fix!
